redis/reply.go

```diff
@@ -317,6 +317,22 @@ func Ints(reply interface{}, err error) ([]int, error) {
 	return ints, nil
 }
 
+
+// Floats is a helper that converts an array command reply to a []float64. If
+// err is not equal to nil, then Floats returns nil, err.
+func Floats(reply interface{}, err error) ([]float64, error) {
+	var floats []float64
+	values, err := Values(reply, err)
+	if err != nil {
+		return floats, err
+	}
+	if err := ScanSlice(values, &floats); err != nil {
```

Comment:
Convert the values directly instead of using ScanSlice.

Reply:
Why I can't use ScanSlice?

Reply:
Write it directly because the code is easier to understand and faster than ScanSlice.

Reply:
Ok, I would change code soon.